readme.md

```diff
@@ -5,16 +5,17 @@
 sudo apt remove cmdtest yarn
 curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
 echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
```

Comment:
Since the corepack is installed by default, we can just `corepack enable` to setup Yarn.

```suggestion
```